lib/browser-specific.js

```diff
@@ -0,0 +1,247 @@
+'use strict';
+
+/**
+ * Implements functionality to report on how many WPT tests fail only on one
+ * browser (aka 'browser-specific failures).
+ */
+
+// A helper class providing an iterator-like interface to either an Object with
+// enumerable properties, or an Array. Iterates in a sorted order; alphabetical
+// in the case of an Object, or in the order defined by |comparatorFunc| in the
+// case of an Array.
```

Comment:
Nit: I'd probably prefer to have a cleaner API by not differentiating arrays and objects, always using `comparatorFunc` to sort and leaving it up to the callers to decide whether a comparator is needed.